examples/example_linearmechanism.m

```diff
@@ -0,0 +1,32 @@
+% Initialization.
```

Comment:
Are there related examples in the official Neuron documentation that we can reference?